### PR TITLE
[#135964665]Update_product

### PR DESCRIPTION
**Pull request by tanjunlem**
I am designing the update product feature into the system for staff to update existing product details.

### CODE REVIEW

File: process/code/Update_Product.txt [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+        jLabel6.setText("Country:");
+
+        jLabel7.setText("Year:");
+
+        jLabel8.setText("Value:");
+
+        jLabel9.setVisible(false);
+        jLabel9.setText("Purity:");
+
+        lblProdID.setText("jLabel10");
```
Comment (TeckYong) [inline process/code/Update_Product.txt:56]: might need to remove unnecessary space for coding.

File: process/code/Update_Product.txt [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+                        .addPreferredGap(javax.swing.LayoutStyle.ComponentPlacement.UNRELATED)
+                        .addGroup(layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
+                            .addGroup(layout.createSequentialGroup()
+                                .addComponent(lblProdID)
+                                .addGap(0, 0, Short.MAX_VALUE))
+                            .addComponent(txtProdName)))
+                    .addGroup(javax.swing.GroupLayout.Alignment.TRAILING, layout.createSequentialGroup()
+                        .addGap(0, 0, Short.MAX_VALUE)
+                        .addComponent(btnUpdate)))
+                .addContainerGap())
```
Comment (MasterODST) [inline process/code/Update_Product.txt]: You may need to correct this format indent errors

Comment (MasterODST) [inline process/code/Update_Product.txt]: You may need to fix the indentation

Comment (MasterODST) [inline process/code/Update_Product.txt:126]: You may need to fix the indentation

**Review by TeckYong — APPROVED** [review]

**Review by chunweng428 — APPROVED** [review]

**Review by MasterODST — APPROVED** [review]

File: process/code/Update_Product.txt (hunk 1 above)
Comment (tanjunlem) [reply process/code/Update_Product.txt:56]: Ya, noted. Thank you.

File: process/code/Update_Product.txt (hunk 2 above)
Comment (tanjunlem) [reply process/code/Update_Product.txt:126]: I think is ok, because the coding is worked as expected. But thank you to remind me.

**Pull request merged**